.github/workflows/main.yml

```
@@ -14,7 +14,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Copy .env
-      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
+      run: php -r "file_exists('.env') || copy('.env', '.env');"
```

надо вернуть файл, так как .env не попадает в репозиторий (и не должен), а вот файл с примером (`.env.example`) должен быть в репозитории
```suggestion
      run: php -r "file_exists('.env') || copy('.env.example', '.env');"
```